package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "conforma-server",
-  "version": "0.5.6-58",
+  "version": "0.5.6-26",
```

Comment:
It's just because I've been making builds from all kinds of different branches, so the version numbers get out of whack on different branches. It's fine though, as long as I reset the number to the last build version when merging everything back to develop.

> Won't this also mean snapshots can't be uploaded to the live server?

The live server is on the latest version (0.5.6-74 as of right now), so that won't be a problem. The number only gets bumped when making a build. Plus, we should be using the latest build branch when working on configs anyway,.